selfdrive/camerad/cameras/camera_qcom2.cc

```diff
@@ -650,23 +648,16 @@ static void camera_open(CameraState *s) {
     .comp_grp_id = 0x0, .split_point = 0x0, .secure_mode = 0x0,
   };
 
-  ret = cam_control(s->multi_cam_state->isp_fd, CAM_ACQUIRE_DEV, &acquire_dev_cmd, sizeof(acquire_dev_cmd));
-  LOGD("acquire isp dev: %d", ret);
+  s->isp_dev_handle = device_acquire(s->multi_cam_state->isp_fd, s->session_handle, &isp_resource).value_or(-1);
```

Comment:
This `value_or` with a magic -1 seems kind of a hack. Think we should make a temporary variable and assert that. It's one more line but should be more clear.
```suggestion
  auto isp_dev_handle = device_acquire(s->multi_cam_state->isp_fd, s->session_handle, &isp_resource);
  assert(isp_dev_handle);
  s->isp_dev_handle = *isp_dev_handle;
```